### ISSUE TITLE
Kerning doesn't show up after conversion to Woff

### ISSUE THREAD

**Issue by HjalmarSnoep** [glyphr-studio/Glyphr-Studio-1#267]
I can see kernpairs in the SVG, but they just don't show up once I convert the font to Woff or Woff2 with online tools.
Might not be an issue of your product, but improved workflow might include a woff export.

**Comment by mattlag** [comment]
Hello! Yes, currently we use Open type.js to read/write OTF files. It has no WOFF support.

The kerning / ligatures support is obviously not ideal, but it's OpenType.js supports at the moment.

I have used https://onlinefontconverter.com/ to convert SVG to OTF, and the kerning / ligatures information survives. 

I'm going to close this because there is nothing really to do, but thank you for the feedback.

**Comment by HjalmarSnoep** [comment]
I solved my own issue by going through SVG and converting that to WOFF. And then the woff to woff2.  With online converters from Might be something for the manual. Forgot to mention it. I agree, there isn't anything to do for you guys, except maybe extending the manual.
If you want me to write a piece about it, I will. I searched quite extensively on the internet, but couldn't find anything about it. Very satisfied user here. Great app!

In a nutshell and in md:
#  how to keep Kerning when converting to webfonts
1. export SVG from glyphr.
2. convert to woff with everythingfonts.com
3. the convert woff to woff2 with [https://everythingfonts.com/woff-to-woff2](everythingfonts.com)
4. Then write the CSS to load the webfonts
5. Also put Kerning and antialiasing on as a feature in CSS.

This seems to work in Chrome, Firefox and Edge!

**Comment by mattlag** [comment]
This is one of the top requests, so it probably deserves it's own help
page.  I would definitely appreciate any writeup you wanted to provide - it
would probably help many others that have this same challenge!  The
nutshell you provided is great, but if you have more detail that would be
good.

Thank you very much!

::Matt

**Comment by HjalmarSnoep** [comment]
I'd love to get more detailed in a tutorial with screenshots and videos and the whole 9-yards.
For me it would be logical in the form of a youtube tutorial or do you prefer another format?
There is no wiki accessible to me for this project on github. Maybe I'm missing something? I'm quite new to github.

I was also thinking, couldn't a back-end solution to the problem help, if the javascript-library doesn't support it? I haven't done any research yet, but I feel a font-parser package like Pomax of PhenX in PHP could possibly solve it. Or do you prefer a purely front-end solution,  because of the server-load?
I that case it might be interesting to port it to javascript? Sounds like another project I might enjoy.
